includes/Admin/Posting.php

```
@@ -543,7 +727,7 @@
             return $post_id;
         }
 
-        list( $post_vars, $tax_vars, $meta_vars ) = self::get_input_fields( $wpuf_cf_form_id );
+        list( $post_vars, $tax_vars, $meta_vars ) = ( new Posting )->get_input_fields( $wpuf_cf_form_id );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$post_vars'. (Unused Code Rules)

(UnusedLocalVariable)